benchmark/test/reference/preconditioner.simple.stdout

```diff
@@ -1,7 +1,6 @@
-
 [
     {
-        "size": 125,
+        "size": 100,
```

Comment:
Yes, before the matrix dimensions were written into the `size` field, now they are being written into `rows` and `cols` to avoid overwriting the input size specified for the stencil.